src/EditorFeatures/Core/Implementation/CodeCleanup/ICodeCleanupService.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.CodeCleanup
+{
+    internal interface ICodeCleanupService : ILanguageService
+    {
+        Task<Document> CleanupDocument(Document document, CancellationToken cancellationToken);
```

Comment:
Task returning methods should end in "Async"

Reply:
Should this just be in the 'Features' project?

Reply:
Why is this under the 'Implementation' namespace?

Reply:
fixed